src/edu/harvard/CS50.java

```diff
@@ -58,6 +58,8 @@
         System.setOut(new PrintStream(new BufferedOutputStream(new FileOutputStream(FileDescriptor.out), 1)));
     }
 
+    private static Scanner s = null;
```

Comment:
For the record, it's only accessed in `getString`. Didn't want to instantiate, unless the user does call one of the `get*` methods?